<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRemoveCustomer</class>
 <widget class="QDialog" name="AddRemoveCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add / Remove Customer</string>
  </property>
  <widget class="QLabel" name="bg_white">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(154, 140, 152);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bg_dark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(74, 78, 105);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="genderComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>F</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="balanceLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="statusComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paid</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Unpaid</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="firstLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="firstLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ageComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="durationLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateLineEdit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lastLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastLineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="durationlabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Duration:</string>
   </property>
  </widget>
  <widget class="QLabel" name="genderLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date Joined:</string>
   </property>
  </widget>
  <widget class="QLabel" name="balanceLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ageLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);
border-radius: 0;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(232, 232, 232);
border-radius: 0;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="addRadioButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Customer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="removeRadioButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remove Customer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="idComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="roomComboBox">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="roomLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Room:</string>
   </property>
  </widget>
  <zorder>bg_dark</zorder>
  <zorder>bg_white</zorder>
  <zorder>genderComboBox</zorder>
  <zorder>balanceLineEdit</zorder>
  <zorder>statusComboBox</zorder>
  <zorder>firstLineEdit</zorder>
  <zorder>firstLabel</zorder>
  <zorder>ageComboBox</zorder>
  <zorder>durationLineEdit</zorder>
  <zorder>dateLineEdit</zorder>
  <zorder>lastLabel</zorder>
  <zorder>lastLineEdit</zorder>
  <zorder>durationlabel</zorder>
  <zorder>genderLabel</zorder>
  <zorder>statusLabel</zorder>
  <zorder>dateLabel</zorder>
  <zorder>balanceLabel</zorder>
  <zorder>ageLabel</zorder>
  <zorder>submitButton</zorder>
  <zorder>backButton</zorder>
  <zorder>addRadioButton</zorder>
  <zorder>removeRadioButton</zorder>
  <zorder>idComboBox</zorder>
  <zorder>roomComboBox</zorder>
  <zorder>roomLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
